library/core/src/main/java/com/google/android/exoplayer2/text/cea/Cea608Decoder.java

```diff
@@ -414,13 +444,17 @@ private void handleCtrl(byte cc1, byte cc2, boolean repeatedControlPossible) {
       }
     }
 
+    if (!isCodeForSelectedChannel(cc1)) {
```

Comment:
Related question: I probably need to understand channels a bit better, but it feels odd that this isn't at least pulled up to the top of this method.

Reply:
The output is probably the same, we can optimize the order during these merges.

Note: There are additional comments in the standard, for example, Frame 2 should avoid repeating commands as it has the same bandwidth but more things to transfer.